epochStart/shardchain/trigger.go

```diff
@@ -454,6 +506,16 @@ func (t *trigger) checkIfTriggerCanBeActivated(hash string, metaHdr *block.MetaB
 	return isMetaHdrFinal, finalityAttestingRound
 }
 
+func (t *trigger) addMissingMiniblocks(missingMiniblocks map[string]*block.MiniBlock) {
+	t.mutMissingMiniblocks.Lock()
+	defer t.mutMissingMiniblocks.Unlock()
+
+	for hash, mb := range missingMiniblocks {
+		t.mapMisingMiniblocks[hash] = mb
+		log.Debug("trigger.addMissingMiniblocks", "hash", []byte(hash))
```

Comment:
Thise are not only missing miniblocks - these are all of them.

Reply:
Now are only the missing ones